backend/api/models/ohshown_event.py

```diff
@@ -13,16 +13,9 @@ class OhshownEvent(SoftDeleteMixin):
 
     # List of fact_type & status
     ohshown_event_type_list = [
-        ("2-1", "沖床、銑床、車床、鏜孔"),
-        ("2-2", "焊接、鑄造、熱處理"),
-        ("2-3", "金屬表面處理、噴漆"),
-        ("3", "塑膠加工、射出"),
-        ("4", "橡膠加工"),
-        ("5", "非金屬礦物（石材）"),
-        ("6", "食品"),
-        ("7", "皮革"),
-        ("8", "紡織"),
-        ("9", "其他"),
+        ("2-1", "目擊黑熊"),
```

Comment:
Looks like it's different (a simplified version) with the UI dropped-down list (according to the screenshot in https://github.com/tai271828/disfactory-backend/issues/15).
Is it good enough for now?

Reply:
And FYI that I have made the dropped-down list in the PR https://github.com/tai271828/disfactory-backend/pull/18.
It was for the get event types API endpoint to synchronize the dropped-down list between the BE and FE.
Maybe we should reuse/integrate this list within this PR.

Reply:
The elements are aligned with the latest frontend drop-down list. See the development site (which is deployed with the latest codebase)

> Looks like it's different (a simplified version) with the UI dropped-down list (according to the screenshot in #15). Is it good enough for now?